src/app/(app)/(home)/search-filters/index.tsx

```
@@ -0,0 +1,15 @@
+import { Categories } from './categories';
+import SearchInput from './search-input';
+
+interface Props {
+  data: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)